examples/mini-compile/src/Main.hs

```diff
@@ -82,7 +82,11 @@ mkDynFlags filename s = do
   let baseFlags =
         (defaultDynFlags fakeSettings fakeLlvmConfig) {
           ghcLink = NoLink
+#if defined(GHC_MASTER)
```

Comment:
You need more CPP here judging from CI. `platormMini_os` is no longer a thing.

Reply:
No it's not. That's a surprise. How are my tests passing locally? I'll look into it.

Reply:
I have the fix to hand. I just need to see if fail here first 😠